### DISCUSSION TITLE
Welcome to Infinite Remote Discussions!

### DISCUSSION THREAD

**Discussion by gitondezhuts** [infiniteremote/installer#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by CybSecUK** [comment]
Much better to manage than rustdesk, well done, keep up the good work,. 
How about the option to see the password for unattended access. or have I missed it ?

**Reply by gitondezhuts** [reply]
Dont think thats been added, shall look into it

**Reply by CybSecUK** [reply]
Thanks, I meant to write set not see :-)
So the installer does all the work

**Reply by gitondezhuts** [reply]
I am hoping we can integrate this https://rdgen.crayoneater.org/

**Reply by BinaryBrother** [reply]
> I am hoping we can integrate this https://rdgen.crayoneater.org/

Woah, nice job. 20-30 mins for generation is a bit much though. Do you need access to more horsepower? This is remarkable.

**Comment by Dekijo86** [comment]
https://rdgen.crayoneater.org/ Hi have you used this site? Is he safe?
